playwright/masonry/two-module.spec.mjs

```diff
@@ -0,0 +1,24 @@
+// @flow strict
+import { expect, test } from '@playwright/test';
+import getServerURL from './utils/getServerURL.mjs';
+import selectors from './utils/selectors.mjs';
+
+test.describe('Masonry: two columns module', () => {
+  test('render the two column module', async ({ page }) => {
+    await page.setViewportSize({ width: 1200, height: 900 });
+    await page.goto(
+      getServerURL({
+        manualFetch: true,
+        twoColItems: true,
+      })
+    );
+
+    // The two modules items appear after the 50th pin, we add at least 60 items
+    const addItemsButton = await page.locator(selectors.addItems);
+    await addItemsButton.click();
+    await addItemsButton.click();
+    await addItemsButton.click();
+
+    await expect(page.getByText('columnSpan: 2')).toBeVisible();
```

Comment:
is there a more useful test we could write here? asserting that the two column module shows up is good but it doesn't do much to help guard against breakages in terms of the actual layout (although the unit test should help with that). 

if this is something you'd prefer to come back to after we do more investigation around optimizing the placement, that's fine too.

Reply:
Yeah, I wanted to do a more precise test but since currently we don't have a way to set a specific set of items on a second layout calculation (have some items first and then add a custom set of items with two column), the two column items are random so I can't set a specific position. I want to add that feature to the masonry performance test but meanwhile this at least test the render part.